gcp/README.md

```diff
@@ -153,6 +153,26 @@ rake sh["gcloud services disable container.googleapis.com"]
 This happens due to limitation of maximum 10 keys per ServiceAccount.
 If you see this error during any `rake` execution, run `rake destroy_sa_keys` and then try again.
 
+### Monitoring and alerting
+
+We use [Stackdriver Beta Monitoring](https://cloud.google.com/monitoring/kubernetes-engine/) to collect various system metrics, navigate through them with [Kubernetes Dashboard](https://app.google.stackdriver.com/kubernetes), and send alerts when they violate thresholds that are being set by [Stackdriver Alerting Policies](https://app.google.stackdriver.com/policies).
+
+Due to the lack of Terraform integration we use [Ruby client](https://github.com/gpii-ops/gpii-infra/blob/master/gcp/modules/gcp-stackdriver-alerting/client.rb) to apply / update / destroy Stackdriver's resource primitives from their [json configs](https://github.com/gpii-ops/gpii-infra/blob/master/gcp/modules/gcp-stackdriver-alerting/resources).
+
+**To add new resource / debug existing resources:**
```

Comment:
Please make this a subheading `####` for better document organization and to provide an anchor to link to.

Reply:
Done.